discord/ext/commands/view.py

```diff
@@ -114,15 +114,34 @@ def quoted_word(view):
     if current is None:
         return None
 
-    is_quoted = current == '"'
+    # map from opening quotes to closing quotes
+    quotes = {
+        '"': '"',
+        '«': '»',
+        '‹': '›',
+        '“': '”',
+        '‘': '’',
+        '❛': '❜',
+        '❝': '❞',
+        '❮': '❯',
+        '〝': '〞',
+        '＂': '＂',
+    }
+
+    is_quoted = current in quotes
+    if is_quoted:
+        open_quote = current
+        close_quote = quotes[open_quote]
+    else:
+        open_quote = close_quote = ''
```

Comment:
This breaks if the input is not actually quoted. Despite its name `quoted_word` takes an *optionally* quoted word.